ui/libs/search/src/lib/collections/trainings/training.model.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from './training-fields.types';
 
 export interface ITraining {
+  id: string;
```

Comment:
Why have you put the id in here, while you're using `Partial<ITraining> & {id: string}`.

Reply:
Because `Partial<ITraining>` can have id or might not have it. The type `Partial<ITraining> & IHasId` WILL have id field which is string.